pegasus/sites.v3/code.org/public/privacy.md

```diff
@@ -188,13 +191,13 @@ Code.org’s officers, directors,employees, agents and contractors must have a l
 
 We do not rent or sell Personal Information, Persistent Identifiers, or any other information that we collect from Users, or exploit it for financial gain in any other way. Code.org will never share or grant rights to Personal Information with other third-party organizations to use without your consent, except as part of a specific program or feature for which you will have the explicit ability and choice to opt-in.
 
-In particular, we do not share any Personal Information you provide with our donors or sponsors (other than the "Public Reporting" of anonymized reports as described below).
+In particular, we do not share any Personal Information you provide with our donors or sponsors (other than the "Public Reporting" of de-identififed reports as described below).
 
-### We may share student achievement data with your school or school district
+### We may share Student achievement data with your school or school district
 
-If your use of the Code.org Website is in the context of a partnership with your school or school district, we may allow the school or school district to access the same student progress report that is also shared with your teacher(s).
+If your use of the Code.org Website is in the context of a partnership with your school or school district, we may allow the school or school district to access the same Student progress report that is also shared with your Teacher(s).
 
-### We may share teacher information with training facilitators and our Regional Partners
+### We may share Teacher information with training facilitators and our Regional Partners
 
 Part of our educational program is to offer professional development workshops to prepare teachers to offer computer science courses. If you are a teacher participating in one of these workshops, your name and contact information will be shared with the facilitator and/or professional learning organization hosting the workshop, and they may have access to your continued progress in our online professional learning courses in order to coach or facilitate your learning. They may also be able to see your overall class progress to support your classroom. They will not have any data on specific students. If Code.org is paying for your travel to our professional learning workshops for teachers, we will - with your explicit permission - share your name and contact information with our travel partner to facilitate booking your travel.
 
```

Comment:
Line 204: employs the `Teacher`, in order .. to compensate `Teachers` in situations ... with their `Teachers`.

Reply:
Line 208

* information of any `Teacher` unless you
* professional development to professional learning